Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue2577.cs

```diff
@@ -49,5 +49,43 @@ public DetailPage()
 				Content = button;
 			}
 		}
+
+#if UITEST
```

Comment:
The test uses specific to the Android methods to search for button
```suggestion
#if UITEST && __ANDROID__
```